CMakeLists.txt

```diff
@@ -210,106 +195,98 @@ if (CMAKE_COMPILER_IS_GNUCC OR CMAKE_C_COMPILER_ID MATCHES "Clang")
     endif()
 endif()
 
-find_library(GLSLANG_LIB NAMES glslang
-             HINTS ${GLSLANG_SEARCH_PATH} )
+set(GLSLANG_INSTALL_DIR "GLSLANG-NOTFOUND" CACHE PATH "Absolute path to a glslang install directory")
+if(NOT GLSLANG_INSTALL_DIR AND NOT DEFINED ENV{GLSLANG_INSTALL_DIR})
+    message(FATAL_ERROR "Must define location of glslang binaries -- see BUILD.md")
+endif()
 
-find_library(OGLCompiler_LIB NAMES OGLCompiler
-             HINTS ${GLSLANG_SEARCH_PATH} )
+# Cmake command line option overrides environment variable
+if(NOT GLSLANG_INSTALL_DIR)
+    set(GLSLANG_INSTALL_DIR $ENV{GLSLANG_INSTALL_DIR})
+endif()
```

Comment:
See my comments about env override in https://github.com/LunarG/VulkanTools/pull/588.  It might need to be added for headers and loader here as well.